pkgs/top-level/all-packages.nix

```
@@ -12593,7 +12593,10 @@ with pkgs;
 
   rnv = callPackage ../tools/text/xml/rnv { };
 
-  roam-research = callPackage ../applications/office/roam-research { };
+  # Using import because otherwise `.override` would override the `default.nix` arguments.
+  roam-research = import ../applications/office/roam-research {
+      inherit stdenv callPackage;
+    };
```

```suggestion
  roam-research = callPackage ../applications/office/roam-research { };
```

See `pkgs/applications/audio/spotify/default.nix` for an example.

---

So I'd have to change default.nix to this?

```{ stdenv, callPackage, ... }@args:
let
  extraArgs = removeAttrs args [ "callPackage" ];
in
if stdenv.isDarwin then
  callPackage ./darwin.nix (extraArgs // { })
else
  callPackage ./linux.nix (extraArgs // { })
```

---

Yes, please.